### DISCUSSION TITLE
Inconsistency in credits fonts

### DISCUSSION THREAD

**Discussion by Mr-Bajs** [ihhub/fheroes2#8069] [General]
### Preliminary checks

- [X] I've checked that there aren't [**other open issues**](https://github.com/ihhub/fheroes2/issues?q=is%3Aissue) on the same topic.

### Describe the problem requiring a solution

I suppose the picture is pretty self explainatory.
![credits](https://github.com/ihhub/fheroes2/assets/93934125/0988c4a4-2147-47f1-a424-f5eeaa43e2e8)

**Comment by ihhub** [comment]
Hi @Mr-Bajs , this is done on purpose as the link is for old project which is not maintained anymore. I am making it as a discussion.
